src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -595,6 +619,16 @@ private IMemberReferenceExpression CreateBoundObjectInitializerMemberOperation(B
                 default:
                     throw ExceptionUtilities.Unreachable;
             }
+
+            IOperation CreateInstanceReferenceExpression()
+            {
+                return new InstanceReferenceExpression(
+                    semanticModel: _semanticModel,
+                    syntax: boundObjectInitializerMember.Syntax,
+                    type: boundObjectInitializerMember.ReceiverType,
+                    constantValue: default(Optional<object>),
+                    isImplicit: boundObjectInitializerMember.WasCompilerGenerated);
```

Comment:
>boundObjectInitializerMember.WasCompilerGenerated [](start = 32, length = 49)

It feels like this node should be always implicit, it doesn't represent anything from source code.

Reply:
I agree. Fixed.